File: Numberof1Bits.cpp
#include<iostream>

using namespace std;

class Solution {
public:
    int hammingWeight(uint32_t n) {
        
        int count = 0;
        
        for(;n>0;n/=2){
            if(n%2 == 1) count++;
        }
        
        return count;
        
    }
};

int main(){
    
    uint32_t n;
    cin>>n;
    
    Solution *solution = new Solution();
    cout<<solution->hammingWeight(n);
    
    return 0;
    
}

File: Numberof1Bits.py
class Solution(object):
    def hammingWeight(self, n):
        """
        :type n: int
        :rtype: int
        """
        count = 0
        while n > 0:
            count += n & 1
            n >>= 1
        return count
        
if __name__ == '__main__':
    n = int(raw_input())
    solution = Solution()
    print(solution.hammingWeight(n))
